nikola/plugins/task/sections.py

```diff
@@ -98,7 +98,7 @@ def get_classification_friendly_name(self, section, lang, only_last_component=Fa
 
     def get_path(self, section, lang, dest_type='page'):
         """Return a path for the given classification."""
-        result = [_f for _f in [section] if _f]
+        result = [_f for _f in [self.site.config['SECTION_PATH'] + section] if _f]
```

Comment:
You need to write `self.site.config['SECTION_PATH'](lang)` to make it translatable.

You can take a look at `tags.py` to see how this could be done correctly:
```
    def get_path(self, classification, lang, dest_type='page'):
        """Return a path for the given classification."""
        return [_f for _f in [
            self.site.config['TAG_PATH'](lang),
            self.slugify_tag_name(classification, lang)] if _f], 'auto'
```
So it should be:
```
        result = [_f for _f in [self.site.config['SECTION_PATH'](lang), section] if _f]
```